Mjml.Net/Components/Body/AccordionElementComponent.cs

```diff
@@ -0,0 +1,142 @@
+using Mjml.Net.Internal;
+using System.Xml;
+
```

Comment:
Can we fix the build warning for consistency?

Reply:
Sure